examples/utils_glue.py

```diff
@@ -442,6 +448,9 @@ def convert_examples_to_features(examples, label_list, max_seq_length,
         # used as as the "sentence vector". Note that this only makes sense because
         # the entire model is fine-tuned.
         tokens = tokens_a + [sep_token]
+        if sep_token_extra:
+            # roberta uses an extra separator b/w pairs of sentences
+            tokens += [sep_token]
```

Comment:
Because we have 3 special tokens for RoBERTa in line 429 should be something like this:
special_tokens_count = 3 if sep_token_extra else 2
if len(tokens_a) > max_seq_length - special_tokens_count:
    tokens_a = tokens_a[:(max_seq_length - special_tokens_count)]

Reply:
I checked and what is funny that it doesn't matter for finetuning task one or two separated tokens you have in the end :) So, I think I will remove extra separated token in the end in my code.

Reply:
You're right about the extra token, thanks for pointing it out!